docs/spec.md

```diff
@@ -367,7 +367,7 @@ Set one or more parameters. Only supported if the server previously declared tha
 - `op`: string `"setParameters"`
 - `parameters`: array of:
   - `name`: string
-  - `value`: number | boolean | string | number[] | boolean[] | string[]
+  - `value`: number | boolean | string | number[] | boolean[] | string[] | undefined. When set to `undefined`, the parameter shall be unset (removed).
```

Comment:
IMO, we should use `null` instead of `undefined` as it is much more explicit that the value is being unset. @jhurliman / @jtbandes what do you think?

Reply:
I think it's dumb that JS has both `null` and `undefined`. Usually I choose to believe I live in a world where `null` does not exist. However, unfortunately JSON only supports `null` and not `undefined`. So actually this spec does not exactly make sense as written... We either need to use null or say that the `value` key can be absent from the JSON.